dev/io.openliberty.http.monitor_fat/fat/src/io/openliberty/http/monitor/fat/BaseTestClass.java

```diff
@@ -195,6 +196,9 @@ protected String requestContainerHttpServlet(String servletPath, String host, in
     }
 
     protected String getVendorMetrics(LibertyServer server) throws Exception {
+
+        TimeUnit.MILLISECONDS.sleep(500);
```

Comment:
Not specifically for the "sum" FATS. Just put it here to avoid any timing issues.